docs/build/guides/conventions/stellar-asset-contract.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: Integrate Stellar Assets Contracts
+description: Test and use Stellar assets in a Soroban smart contract
+---
+
+From the contract perspective Stellar Asset Contract is not different from any other token that implements the Soroban [SEP-41 Token Interface]. The Rust SDK contains a pregenerated client for any contract that implements the token interface:
+
+```rust
+use soroban_sdk::token;
+
+struct MyContract;
+
+#[contractimpl]
+impl MyContract {
+  fn token_fn(e: Env, id: Address) {
+    // Create a client instance for the provided token identifier. If the id
+    // value corresponds to an SAC contract, then SAC implementation is used.
+    let client = token::TokenClient::new(&env, &id);
+    // Call token operations part of the SEP-41 token interface
+    client.transfer(...);
+  }
+}
+```
+
+The address parameter is not the address of the issuer of an asset, it corresponds to the deployed contract address for this asset.
+
+```bash
+stellar contract id asset \
+    --source G... \
+    --network testnet \
+    --asset [asset:issuer]
+```
+
+E.g. for USDC, it would be `--asset USDC:G...` For the native asset, XLM, `--asset native`. See the [deploy SAC] guide for more details.
+
+[deploy SAC]: ../cli/deploy-stellar-asset-contract.mdx
+
+:::info Clients
```

Comment:
```suggestion
:::info[Clients]
```

As part of the upgrade from docusaurus v2 to v3, the way we do [admonition titles changes](https://docusaurus.io/docs/migration/v3#admonitions-option). There is a compatibility flag that is turned on by default, but we'll want to migrate all existing admonition titles to use the new syntax. So, for new ones, let's use the new way to begin.

Reply:
A great thanks!